frontend/public/components/_quota.scss

```diff
@@ -5,3 +5,29 @@
 .co-resource-quota-empty {
   font-size: 15px;
 }
+
+.co-resource-quota-chart-row {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 200px;
+  padding-right: 200px;
```

Comment:
These padding values would be way too large for mobile.... Why are they needed if we're already centering?

Reply:
Ok, removed padding, wasn't actually centering.  Added `justify-content: center`